.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
   "extends": ["ts-react-important-stuff", "plugin:prettier/recommended"],
-  "parser": "@babel/eslint-parser"
+  "parser": "@babel/eslint-parser",
+  "rules": {
+    "prettier/prettier": "warn"
+  }
```

Comment:
This is a personal preference and accidentally included it in this PR. It marks all ES Lint concerns at warning level in the IDE (instead of error).